src/lib/webext.ts

```diff
@@ -26,14 +26,8 @@ export function getContext() {
     }
 }
 
-export let browserBg
-
 // Make this library work for both content and background.
-if (inContentScript()) {
-    browserBg = browserProxy
-} else {
-    browserBg = browser
-}
+export const browserBg = inContentScript() ? browserProxy : browser
```

Comment:
This is sort of unrelated, but improves coverage of types a lot as the type is inferred correctly now